v21.2/restoring-backups-across-versions.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Restoring Backups Across Versions
+summary: CockroachDB's support policy for restoring backups across versions.
+toc: true
+docs_area: manage
+---
+
+This page describes the support Cockroach Labs provides for restoring backups across major versions of CockroachDB. The sections outline the support and examples for the most common use cases when restoring backups across versions:
+
+- [Support for restoring backups into a higher version](#support-for-restoring-backups-into-a-higher-version)
+- [Restore into a higher version](#restore-into-a-higher-version)
+- [Support for long-term backup archival](#support-for-long-term-backup-archival)
+
+## Support for restoring backups into a higher version
+
+Cockroach Labs supports restoring a backup taken on a cluster on a specific major version into a cluster that is on the next major version. Therefore, if you took a backup on major version N, you can restore it on major version N+1.
```

Comment:
maybe this is too obvious to need to state, but should we say "on the same version _or_  the next major..."?

Reply:
Good catch, worth stating the obvious.